examples/simple-genai-server/README.md

```diff
@@ -37,45 +37,17 @@ Follow the instructions for
 
 ### Set the Game Servers on the Same Cluster as the GenAI Inference Server
```

Comment:
I'd suggest leaving a very short description on the README and then adding a link to point folks at the website instead of duplicating all of the content in both places.